examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java

```diff
@@ -0,0 +1,155 @@
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.EncryptionContext;
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.EncryptionFlags;
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.providers.DirectKmsMaterialProvider;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.AWSKMSClientBuilder;
+
+import java.security.GeneralSecurityException;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static com.amazonaws.services.dynamodbv2.datamodeling.encryption.utils.EncryptionContextOperators.overrideEncryptionContextTableNameUsingMap;
+
+public class EncryptionContextOverridesWithDynamoDBMapper {
+    public static void main(String[] args) throws GeneralSecurityException {
+        final String cmkArn = args[0];
+        final String region = args[1];
+        final String encryptionContextTableName = args[2];
+
+        AmazonDynamoDB ddb = null;
+        AWSKMS kms = null;
+        try {
+            ddb = AmazonDynamoDBClientBuilder.standard().withRegion(region).build();
+            kms = AWSKMSClientBuilder.standard().withRegion(region).build();
+            encryptRecord(cmkArn, encryptionContextTableName, ddb, kms);
+        } finally {
+            if (ddb != null) {
+                ddb.shutdown();
+            }
+            if (kms != null) {
+                kms.shutdown();
+            }
+        }
+    }
+
+    public static void encryptRecord(final String cmkArn,
+                                     final String newEncryptionContextTableName,
+                                     AmazonDynamoDB ddb,
+                                     AWSKMS kms) throws GeneralSecurityException {
+        // Sample object to be encrypted
+        ExampleItem record = new ExampleItem();
+        record.setPartitionAttribute("is this");
+        record.setSortAttribute(55);
+        record.setExample("my data");
+
+        // Set up our configuration and clients
+        final DirectKmsMaterialProvider cmp = new DirectKmsMaterialProvider(kms, cmkArn);
+        final DynamoDBEncryptor encryptor = DynamoDBEncryptor.getInstance(cmp);
+
+        Map<String, String> tableNameEncryptionContextOverrides = new HashMap<>();
+        tableNameEncryptionContextOverrides.put("ExampleTableForEncryptionContextOverrides", newEncryptionContextTableName);
+        tableNameEncryptionContextOverrides.put("AnotherExampleTableForEncryptionContextOverrides", "this table doesn't exist");
+
+        // Supply an operator to override the table name used in the encryption context
+        encryptor.setEncryptionContextOverrideOperator(
+                overrideEncryptionContextTableNameUsingMap(tableNameEncryptionContextOverrides)
+        );
+
+        // Mapper Creation
+        // Please note the use of SaveBehavior.CLOBBER (SaveBehavior.PUT works as well).
```

Comment:
We should encourage people towards `PUT` as it is almost always the better option.